security/cmd/istio_ca/main.go

```diff
@@ -141,6 +142,12 @@ func runCA() {
 	stopCh := make(chan struct{})
 	sc.Run(stopCh)
 
+	serviceController := controller.NewServiceController(cs.CoreV1(), opts.namespace,
+		func(*v1.Service) {},
```

Comment:
Is this WIP as all the handlers are empty?

Also it'd be strange if eventually all the handlers are defined here in `main.go`.

Reply:
I want to submit controller and registry in two PRs so I added an empty controller here.

Yes it is. In next PR registry will be created in its own package and the handlers are defined there.